src/store/favorites/logic.js

```diff
@@ -41,11 +38,10 @@ const getFavoritesLogic = createLogic({
 
   async process({ httpClient, action }, dispatch, done) {
     try {
-      const SSID = Cookies.get('SESSION_ID');
       const page = action.payload;
       const { data } = await httpClient({
         method: 'get',
-        url: `account/{account_id}/favorite/movies?api_key=${API}&session_id=${SSID}&sort_by=created_at.asc&page=${page}`,
+        url: `account/{account_id}/favorite/movies?&sort_by=created_at.asc&page=${page}`,
```

Comment:
You don't need & before sort_by param, also it's better to pass params as second argument as object